src/services/member/schemas.ts

```diff
@@ -240,7 +281,7 @@ export const updateOne = {
   params: entityIdSchemaRef,
   body: updateMemberRequiredOneSchemaRef,
   response: {
-    [StatusCodes.OK]: currentMemberSchemaRef,
+    [StatusCodes.OK]: currentAccountSchemaRef,
```

Comment:
I am thinking that the frontend usually does not make use of the return values from POST and PATCH calls (unless we use `mutateAsync` so maybe we could spare the returns and rely on the stale fetching from the hooks ? since typeorm is also not very useful when we want to get the value we have to perform a get after the update. Maybe something to think about for later.